crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -191,6 +191,10 @@ impl<'a> Visitor<'a> {
 
             debug!("task {} hash is {}", info, task_hash);
             if self.dry {
+                self.run_tracker
+                    .track_task(info.clone().into_owned())
```

Comment:
Tiniest nit: we don't need the `into_owned` call here since it's already a `TaskId<'static>`
```suggestion
                    .track_task(info.clone())
```